source/java-drivers.txt

```diff
@@ -9,7 +9,7 @@ Java MongoDB Drivers
 .. toctree::
    :titlesonly:
 
-   /java.txt
+   Java Driver <https://docs.mongodb.com/drivers/java/sync/current/>
```

Comment:
Does 
```
:driver:`Java Driver </java/sync/current>`
```
 work in the toctree?

Reply:
It doesn't appear to in testing :/

Reply:
Ah, I guess it was "won't do": https://jira.mongodb.org/browse/DOP-815